### PR TITLE
feat: show io urls & flyteremote usage

### PR DESCRIPTION
**Pull request by ursucarina**
# TL;DR
* Shows Context Panel IO URLs
* Shows Context Panel IO FlyteRemote usage
* Shows Context Panel node execution logs

|        |           |
| ------------- |:-------------:| 
| image| image |
| image | image |

## Type
 - [ ] Bug Fix
 - [x] Feature
 - [ ] Plugin

## Are all requirements met?

 - [x] Code completed
 - [ ] Smoke tested
 - [ ] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description
 _How did you fix the bug, make the feature etc. Link to any design docs etc_

## Tracking Issue
_Remove the '*fixes*' keyword if there will be multiple PRs to fix the linked issue_

fixes https://github.com/flyteorg/flyte/issues/<number>

## Follow-up issue
_NA_
OR
_https://github.com/flyteorg/flyte/issues/<number>_

### CODE REVIEW

**Comment by jpvotta** [comment]
@pingsutw how does this work with maptask (subtasks)? I believe we are doing some parsing in the FE in order to allocate the input/output list elements into inputs/outputs of each subtask respectively. I am guessing that we would want to include these new visual elements in the maptask parent (since the literalmap will include the full list of inputs/outputs including all subtasks), but not for each subtask (since this would be identical to the info in the parent).

**Comment by pingsutw** [comment]
> I believe we are doing some parsing in the FE in order to allocate the input/output list elements into inputs/outputs of each subtask respectively.

Right, so there is no flyte url for sub task. we can just show it only in the parent for now.
btw, UI looks pretty good now, thank you so much.

one nit: we should not show the copy URL and code snippet if the input or output is empty.

**Comment by pingsutw** [comment]
cc @wild-endeavor

**Review by FrankFlitton — APPROVED** [review]
